person.rb

```diff
@@ -0,0 +1,22 @@
+class Person
+  attr_reader :id
+  attr_accessor :name, age
+
+  def initialize(type, age, _parent_permission, name = 'unkown')
+    @id = Random.rand(1..1000)
+    @name = name
+    @age = age
+    @type = type
+    @parent_permission = true
+  end
```

Comment:
- [ ] Your `can_use_services?` will always return `true` because of this line of code. The essence is to make the parents have an option for deciding if a student that is under 18 years have access to certain services. To achieve this, you can set it in the parameter like this: `def initialize(age, parent_permission = true, name = 'unkown')`. This way, you can set the `parent_permission` to false if you don't want a student that is below 18 to have access to certain services. You can read this [article](https://github.com/microverseinc/curriculum-ruby/blob/main/oop/articles/class_and_object_for_irb.md#constructor-and-instance-vars) for clarification 👍